internal/seetie/convert.go

```diff
@@ -85,10 +85,7 @@ func ValuesMapToValue(m url.Values) cty.Value {
 }
 
 func ListToValue(l []interface{}) cty.Value {
-	if len(l) == 0 {
-		return cty.NilVal
-	}
-	var list []cty.Value
+	list := []cty.Value{}
```

Comment:
Range over an empty slice is the same as doing so with a nil slice. But with allocation.
I am fine with returning a nil value slice instead of NilVal.

just `var list []cty.Value` should do the trick.